assets/css/admin.scss

```diff
@@ -753,9 +753,10 @@
 	}
 
 	.storefront {
+		max-width: 992px;
```

Comment:
This should be 990px since the main container is 1032px width and with 992px and the paddings and border it adds up to 1034px.

Reply:
Good catch!